apps/dialtone-documentation/docs/.vuepress/baseComponents/CodeExampleTabs.vue

```diff
@@ -40,7 +40,8 @@
         kind="warning"
         hide-close
       >
-        This component needs Javascript to work as the example
+        When using HTML / CSS code only the visuals of the component are rendered. It may require
+        additional javascript to function the same way as the example.
```

Comment:
We'll figure out a better way of saying this, though don't have to solve it for this PR.

Reply:
Haha yeah I rewrote this like 5 times and it still didn't sound right